src/Row.js

```diff
@@ -156,7 +156,7 @@ export default class Row extends Component {
     return (
       <Animated.View
         {...this._panResponder.panHandlers}
-        style={[style, styles.container, this._animatedLocation.getLayout()]}
+        style={[style, styles.container, {transform: this._animatedLocation.getTranslateTransform()}]}
```

Comment:
What are you testing with? Testing on an actual Android device shows significant improvements for me. I can't find it documented anywhere why it's better, but it seems to be the case. It's not very noticeably better on iOS though, but Animated has been better for iOS in my experience.